packages/@react-spectrum/menu/docs/MenuTrigger.mdx

```diff
@@ -0,0 +1,516 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/menu';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+```jsx import
+import {ActionButton} from '@react-spectrum/button'
+import Copy from '@spectrum-icons/workflow/Copy';
+import Cut from '@spectrum-icons/workflow/Cut';
+import {Item, Menu, MenuTrigger, Section} from '@react-spectrum/menu';
+import Paste from '@spectrum-icons/workflow/Paste';
+```
+
+---
+category: Collections
+---
+
+# MenuTrigger and Menu
+
+<p>
+  {docs.exports.MenuTrigger.description}
+  <br />
+  {docs.exports.Menu.description}
+</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['Menu', 'MenuTrigger']}
+  sourceData={[
+    {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/popovers/'}
+  ]} />
+
+## Example
+
+```tsx example
+<MenuTrigger>
+  <ActionButton>
+      Edit
+  </ActionButton>
+  <Menu>
+    <Item>Cut</Item>
+    <Item>Copy</Item>
+    <Item>Paste</Item>
+  </Menu>
+</MenuTrigger>
+```
+
+## Content
+
+The Menu accepts Items and [`Sections`](#sections) as children. Items can be
+statically populated (initial example above) or dynamically (below). The dynamic
+method would be better suited to use if the actions within a Menu came from a
+data object such as values returned from an API call. The `uniqueKey` prop needs
+to be set on an Item when statically defining Items and the `itemKey` prop in
+the Menu when its Items are dynamically populated.
+
+```tsx example
+<MenuTrigger>
+  <ActionButton>
+      Edit
+  </ActionButton>
+  <Menu
+    items={[{name: 'Cut'}, {name: 'Copy'}, {name: 'Paste'}]}
+    itemKey="name">
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+```
+
+The MenuTrigger accepts exactly two children: the Menu and the element which
```

Comment:
Once I removed the example for this section because it mirrored the initial value I wonder if these two sections should go above the dynamic example. It feels slightly off, but I'm leaning towards no.

Reply:
I think how you have it is fine as is